src/ArtDetails/ArtDetails.tsx

```diff
@@ -3,12 +3,18 @@ import './ArtDetails.scss';
 import '../ArtView/ArtView.scss';
 import { Link } from 'react-router-dom'
 import { ArtDetailsProps } from '../interface';
+import { motion } from "framer-motion"
 
 const ArtDetails: React.FC<ArtDetailsProps> = props => {
   const likedMessage = props.isLiked ? <p className="percent-message">You and {Math.floor(Math.random() * 100) + 1 + '%'} of other voters liked this art! You have great taste!</p>
                                      : <p className="percent-message">You and {Math.floor(Math.random() * 100) + 1 + '%'} of other people agree! Congratulations!</p>
   return (
-    <section className="art-details-container">
+    <motion.section 
+      className="art-details-container"
+      animate={{ opacity: 1 }}
+      initial={{ opacity: 0 }}
+      transition={{ duration: .4 }}
+    >
```

Comment:
Really great addition to our application! Thank you for implementing this feature!